src/app/header/header.component.html

```diff
@@ -0,0 +1,18 @@
+<header id="head">
+    <div class="logo"></div>
+    <div>
+        <nav>
+            <ul class="divnav">
+                <li><a [routerLink]="['/home']">Accueil</a></li>
+                <li><a class="categ" [routerLink]="['/categ']">Catégories</a></li>
+                <li><a id="fun" [routerLink]="['/fun']">Hav'Fun</a></li>
```

Comment:
correct path would be categories/fun ?

Reply:
The routing will arrive after the merge of this branch and the dev. These routing are temporaries